app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -1,14 +1,12 @@
 module Api
   module V0
     class ClassifiedListingsController < ApiController
+      include Pundit
       include ClassifiedListingsToolkit
 
-      respond_to :json
```

Comment:
Does this get us any kind of performance benefit or is it more of a code cleanup?

Reply:
nope, no benefits, i just moved it to the parent controller to avoid repeating it each time in each controller